c/meterpreter/source/extensions/extapi/clipboard.c

```diff
@@ -257,6 +257,27 @@ DWORD initialise_clipboard()
 	return dwResult;
 }
 
+/*!
+ * @brief Attempt to open the clipboard. If opening the clipboard fails, it will be retried until success or failure.
```

Comment:
```suggestion
 * @brief Attempt to open the clipboard. If opening the clipboard fails, it will be retried until max_retry_count is hit, or success occurs.
```